platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/internal/EventImpl.java

```diff
@@ -542,6 +542,12 @@ public long getGeneration() {
         return baseEvent.getHashedData().getGeneration();
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public long getBirthRound() {
+        return baseEvent.getHashedData().getBirthRound();
+    }
```

Comment:
You told me to take this function out of a PR of mine the other week. 🤣 I don't mind it being here if it's convenient though.

Reply:
lol, sorry about that.  Many apologies.